sys-apps/dstat/dstat-0.7.4.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_6} )
```

Comment:
Any chance you could try with python3.7?

Reply:
3.7 looks fine and trying it picked up the missing dep on six and the collections deprecation warning.